src/util/filteringStream.ts

```diff
@@ -0,0 +1,36 @@
+import { Writable } from 'stream';
+import {middlewareLoggerName} from "../middleware/log-middleware";
+
+/**
+ * Creates a writable stream that prevents HTTP logs from being logged
+ *
+ * It is done to filter out HTTP logs which are coming from Probot.
+ * Unfortunately there is no other way to disable these logs.
+ *
+ * @name filteringStream
+ * @function
+ * @param out {Stream} original stream
+ * @return {WritableStream} that you can pipe bunyan output into
+ */
+const filteringStream = (out) => {
+  const writable = new Writable({
+    write: function(chunk, encoding, next) {
+      if (!shouldBeFiltered(chunk)) {
+        out.write(chunk, encoding)
+      }
+      next();
+    }
+  });
+  return writable
+}
+
+//TODO Remove this code when there will be convenient way to do it in Probot.
+//See https://github.com/probot/probot/issues/1577
+const shouldBeFiltered = (chunk: any) : boolean => {
+  return chunk.includes && (chunk.includes("GET /")
```

Comment:
This could be simplified as 
```
return chunk.toString().match(`${middlewareLoggerName}.*(GET|POST|DELETE|PUT|PATCH) \/`)
```

Reply:
Hence every single log in our app will come through this check, I was trying to avoid creating extra objects during this check. So that's why I used only methods Buffer provided me. 

Maybe it will be OK to spawn an extra string here, but I'd prefer to stay safe. Logging is in general is pretty resource-intensive part of a service

Reply:
creating a string from a buffer will be quicker than doing a loop of the buffer multiple times for the same logic.  The string creation from the buffer is done at the node vm level, as should be very quick.

Reply:
Ok, in thus case I agree